hive/upload_new_hive_version.sh

```diff
@@ -34,15 +39,15 @@ cd "$WORK_DIR" || exit
 bin_file="apache-hive-${VERSION}-bin.tar.gz"
 src_file="apache-hive-$VERSION-src.tar.gz"
 
-echo "Downloading Hive (this can take a while, it is intentionally downloading from a slow mirror that contains all old versions)"
-curl --fail -LOs "https://dlcdn.apache.org/hive/hive-${VERSION}/${bin_file}"
-curl --fail -LOs "https://dlcdn.apache.org/hive/hive-${VERSION}/${bin_file}.asc"
-curl --fail -LOs "https://dlcdn.apache.org/hive/hive-${VERSION}/${bin_file}.sha256"
+echo "Downloading Hive (if this fails, try switching the BASE_DOWNLOAD_URL to the archive)"
+curl --fail -LO --progress-bar "${BASE_DOWNLOAD_URL}/hive-${VERSION}/${bin_file}"
+curl --fail -LO --progress-bar "${BASE_DOWNLOAD_URL}/hive-${VERSION}/${bin_file}.asc"
+curl --fail -LO --progress-bar "${BASE_DOWNLOAD_URL}/hive-${VERSION}/${bin_file}.sha256"
```

Comment:
suggestion: I think we should mention _what_ we download.

```suggestion
echo "Downloading Hive Binary (if this fails, try switching the BASE_DOWNLOAD_URL to the archive)"
curl --fail -LO --progress-bar "${BASE_DOWNLOAD_URL}/hive-${VERSION}/${bin_file}"
curl --fail -LO --progress-bar "${BASE_DOWNLOAD_URL}/hive-${VERSION}/${bin_file}.asc"
curl --fail -LO --progress-bar "${BASE_DOWNLOAD_URL}/hive-${VERSION}/${bin_file}.sha256"
```